python/lsst/ts/externalscripts/auxtel/build_pointing_model.py

```diff
@@ -131,8 +132,10 @@ def get_schema(cls):
             magnitude_limit+magnitude_range.
     datapath:
         type: string
-        default: /project/shared/auxTel
-        description: Path to the data butler where the images can be located.
+        default: /repo/LATISS/
+        description: >-
+            Path to the gen3 butler data repository where the images are located.
```

Comment:
@tribeiro - you requested that I put in gen3. Personally I'm ok with leaving it, especially during the transition. We can rip it out later if it ever causes confusion.

Reply:
Fair enough, just was pointing out that that'll be soon (and note that you only have to pass `repoIsGen3` is you're instantiating with a path rather than with a butler, which I've been discouraging all over this review 😉)

Reply:
decided to leave it for now, so marking this (and the ones below) as resolved.